autogpt_platform/backend/backend/executor/scheduler_unit_test.py

```
@@ -166,7 +171,13 @@ def test_job_to_info_unparseable_returns_none():
 def _patched_experts_db(get_expert: AsyncMock):
     accessor = MagicMock()
     accessor.return_value.get_expert = get_expert
-    return patch(f"{_SCHEDULER_PATH}.experts_db", new=accessor)
+    # The scope gate consults the visibility-blind existence probe before
+    # ruling "missing"; default it to False so get_expert=None means the
+    # row is truly gone.
+    accessor.return_value.expert_row_exists = AsyncMock(return_value=False)
+
+
+return patch(f"{_SCHEDULER_PATH}.experts_db", new=accessor)
```

Ruff (0.16.1)

[error] `return` statement outside of a function/method

(F706)